docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,25 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server is used to expose metrics and CMOS Prometheus server is used to scrape those metrics.
+
+== Overview
+
+https://prometheus.io/[Prometheus^] is a leading open-source monitoring solution.
+As a https://www.cncf.io/projects/prometheus/[graduated project^] of the Cloud Native Computing Foundation, it has become the industry standard for metrics collection and alert generation on cloud platforms.
+
+Monitoring Couchbase is a critical aspect of production use case.
+Couchbase exposes detailed metrics via its own prometheus server starting with Couchbase Server 7+ that can be collected for visualization and other forms of monitoring.
+CMOS prometheus server can be used to scrape these metrics to develop alerts for system issues, identify maintenance windows and other valuable business intelligence insights.
+
+image::component-prometheus.png[]
```

Comment:
This will need the bugfix for Github rendering: 
```
ifdef::env-github[]
:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
endif::[]
```
Otherwise the images will not appear correctly in Github.